app/templateusage/index/controller.js

```diff
@@ -0,0 +1,44 @@
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import Controller from '@ember/controller';
+
+export default Controller.extend({
+  userSettings: service(),
+  theme: service('emt-themes/ember-bootstrap-v5'),
+  timestampPreference: null,
+  template: service(),
+  store: service(),
+
+  pipelineMetrics: computed('routeParams', {
```

Comment:
What we are fetching list of pipelines using the template. There are no metrics retrieved.
```suggestion
  pipelines: computed('routeParams', {
```

Reply:
Not addressed yet